RHEL/6/input/fixes/bash/disable_ctrlaltdel_reboot.sh

```diff
@@ -0,0 +1,4 @@
+
+sed -i 's,^exec.*,exec /usr/bin/logger -p authpriv.notice -t init "Ctrl-Alt-Del was pressed and ignored",'  FILE
```

Comment:
@shawndwells `FILE` should be `/etc/init/control-alt-delete.override` and should probably have a `cp -p /etc/init/control-alt-delete.conf /etc/init/control-alt-delete.override` before the `sed` command.